### DISCUSSION TITLE
Confusion around the Global Component

### DISCUSSION THREAD

**Discussion by dkristensen** [mantinedev/mantine#1231] [Q&A] [answered]
I'm struggling to understand how to use the Global Component to set global styling by reading the docs.

Is this a component I call around my App component like this? Or should I be wrapping my base `Layout` container for each page?

```
// pages/_app.tsx
<Global styleCodeHere>
   <MantineProvider params>
      <Component props>
   </MantineProvider>
</Global>

// This way ^ or this way v

// Containers/Layout.ts
return(
<Global styleCodeHere>
    <MyNavBar/>
        {props.children}
    <MyFooter/>
</Global>
);
```

I've been racking my brain on this for days thinking it's probably just a simple thing I'm missing or overlooking in the documentation, but I'd really appreciate any help on this one!

**Comment by rtivital** [comment, marked as answer]
Global component should be rendered as a separate component somewhere inside your MantineProvider, it should not have any children.

```
<Global styles={{}} />
<App />
```

**Reply by dkristensen** [reply]
Ah, thank you!
